package.json

```diff
@@ -14,18 +14,23 @@
     "@types/styled-components": "^5.1.11",
     "babel-loader": "^8.2.2",
     "electron": "^13.1.6",
+    "electron-builder": "^22.11.7",
     "ts-loader": "^9.2.3",
     "typescript": "^4.3.5",
     "webpack": "^5.44.0",
     "webpack-cli": "^4.7.2",
     "webpack-serve": "^4.0.0"
   },
   "scripts": {
-    "build": "webpack --config webpack.react.config.js --config webpack.electron.config.js",
-    "build:watch": "webpack --config webpack.react.config.js --config webpack.electron.config.js --watch",
-    "start": "electron ./dist/electron/index.js",
+    "build": "yarn run buildcli && webpack --config webpack.react.config.js --config webpack.electron.config.js",
```

Comment:
I'm not sure I want to build the SFE everytime I want to rebuild the react/electron app. Can we have something seperate for those when in dev and maybe something unified for building the final product?

Reply:
Unless there is another command I should be using?

Reply:
No, you're right.  Meant to fix this before checking in, consider it done.